server/files/javascript/icon.js

```diff
@@ -45,6 +45,23 @@ semantic.icon.ready = function() {
       var viewHeight = Math.max(document.documentElement.clientHeight, window.innerHeight);
       return !(rect.bottom < 0 || rect.top - viewHeight >= 0);
     },
+    iconVisible = function (icon) {
+      var rect = icon.getBoundingClientRect();
+      var viewHeight = Math.max(document.documentElement.clientHeight, window.innerHeight);
+      return !(rect.bottom < 0 || rect.top - viewHeight >= 0);
+    },
+    checkIconVisibility = function () {
+      $('.icon.example')
+        .find('i.icon')
+        .each(function () {
+          var element = $(this).get(0);
+          if (iconVisible(element)) {
+            $(this).css('visibility', 'visible');
+          } else {
+            $(this).css('visibility', 'hidden');
+          }
+        });
+    },
     checkVisibility = function() {
```

Comment:
The previous function is now not used anymore. Any reason you kept left it in?

Reply:
Fixed.